src/styles/blocks/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(./../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
+}
+
+.star:first-child {
+  margin-left: 0;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
```

Comment:
extra actions
better to set, for example, right margin, and then remove it for the last child only